json-endpoints/jetpack/class.jetpack-json-api-themes-install-endpoint.php

```diff
@@ -48,6 +48,18 @@ protected function validate_themes() {
 			return new WP_Error( 'missing_themes', __( 'No themes found.', 'jetpack' ) );
 		}
 		foreach( $this->themes as $index => $theme ) {
+			$prefixed_theme = explode( 'wpcom:', $theme );
+			if ( $prefixed_theme ) {
```

Comment:
I think this part would always be true. I think you want to use something like `wp_startswith('wpcom:', $theme )`